Lib/_pydatetime.py

```diff
@@ -1101,16 +1110,18 @@ def __format__(self, fmt):
             return self.strftime(fmt)
         return str(self)
 
-    def isoformat(self):
+    def isoformat(self, basic=False):
         """Return the date formatted according to ISO.
 
-        This is 'YYYY-MM-DD'.
+        This is 'YYYY-MM-DD' or 'YYYYMMDD' if *basic* is true.
 
         References:
         - http://www.w3.org/TR/NOTE-datetime
         - http://www.cl.cam.ac.uk/~mgk25/iso-time.html
```

Comment:
Maybe we can update these while we are modifying this docstring?

```suggestion
        - https://www.w3.org/TR/NOTE-datetime
        - https://www.cl.cam.ac.uk/~mgk25/iso-time.html
```

(They explain both formats :-)

Reply:
Let's do it.